lighthouse/validators/samples_declarations.py

```diff
@@ -88,3 +90,48 @@ def pre_samples_declarations_post_callback(request):
 
     return request
 
+
+def build_clean_elems_object(items, request):
+    i = 0
+    clean_elems = {}
+    while i < len(items):
+        if items[i]["_status"] == "OK":
+            clean_elems[i] = request.json[i]
+        i = i + 1
+    return clean_elems
+
+
+def merge_response_into_payload(payload, response, clean_elems):
+    pos = 0
+    keys = list(clean_elems.keys())
+
+    if len(keys) == 1:
+        payload.json["_items"][keys[0]] = response
+    else:
+        while pos < len(keys):
+            key = keys[pos]
+            payload.json["_items"][key] = response["_items"][pos]
+            pos = pos + 1
```

Comment:
Is it necessary to treat it differently when there's only 1 'clean_elem' because the response format is different? (no "_items" element?)

Reply:
yes, the original response created doesnt have an _item attribute, instead it has all in the root of json so I was treating it different, I think I'm missing a lot of comments. Sorry for that. I'm changing it.